python/hail/methods/io.py

```diff
@@ -0,0 +1,219 @@
+    invalid motifs in the ``contig:start-end`` format here.
+
+    Parameters
+    ----------
+    filename : :obj:`str`
+        Path to file.
+
+    reference_genome : :class:`.GenomeReference`
+        Reference genome to use. Default is
+        :class:`.HailContext.default_reference`.
+
+    Returns
+    -------
+    :class:`.Table`
+        Interval-keyed table.
+    """
+
+    rg = reference_genome if reference_genome else Env.hc().default_reference
+    t = Env.hail().table.Table.importIntervalList(Env.hc()._jhc, path, rg._jrep)
+    return Table(t)
+
+@handle_py4j
+@typecheck(path=strlike,
+           reference_genome=nullable(GenomeReference))
+def import_bed(path, reference_genome=None):
+    """Import a UCSC .bed file as a key table.
+
+    Examples
+    --------
+
+    Add the variant annotation ``va.cnvRegion: Boolean`` indicating inclusion in
+    at least one interval of the three-column BED file `file1.bed`:
+
+    >>> bed = methods.import_bed('data/file1.bed')
+    >>> vds_result = vds.annotate_rows(cnvRegion = bed[vds.v])
+
+    Add a variant annotation **va.cnvRegion** (*String*) with value given by the
+    fourth column of ``file2.bed``:
+
+    >>> bed = methods.import_bed('data/file2.bed')
+    >>> vds_result = vds.annotate_rows(cnvID = bed[vds.v])
+
+    The file formats are
+
+    .. code-block:: text
+
+        $ cat data/file1.bed
+        track name="BedTest"
+        20    1          14000000
+        20    17000000   18000000
+        ...
+
+        $ cat file2.bed
+        track name="BedTest"
+        20    1          14000000  cnv1
+        20    17000000   18000000  cnv2
+        ...
+
+
+    Notes
+    -----
+
+    The key table produced by this method has one of two possible structures. If
+    the .bed file has only three fields (``chrom``, ``chromStart``, and
+    ``chromEnd``), then the produced key table has only one column:
+
+        - **interval** (*Interval*) - Genomic interval.
+
+    If the .bed file has four or more columns, then Hail will store the fourth
+    column in the table:
+
+        - **interval** (*Interval*) - Genomic interval.
+        - **target** (*String*) - Fourth column of .bed file.
+
+    `UCSC bed files <https://genome.ucsc.edu/FAQ/FAQformat.html#format1>`__ can
+    have up to 12 fields, but Hail will only ever look at the first four. Hail
+    ignores header lines in BED files.
+
+    Warning
+    -------
+        UCSC BED files are 0-indexed and end-exclusive. The line "5  100  105"
+        will contain locus ``5:105`` but not ``5:100``. Details
+        `here <http://genome.ucsc.edu/blog/the-ucsc-genome-browser-coordinate-counting-systems/>`__.
+
+    Parameters
+    ----------
+    path : :obj:`str`
+        Path to .bed file.
+
+    reference_genome : :class:`.GenomeReference`
+        Reference genome to use. Default is
+        :meth:`.HailContext.default_reference`.
+
+    Returns
+    -------
+        :class:`.Table`
+        """
```

Comment:
Return type should be aligned with the Returns header. Add a description, though.